### PR TITLE
Fixed #26983 -- Fixed isnull filtering on ForeignKey with to_field

### PR DESCRIPTION
**Pull request by claudep**
Thanks weidwonder for the report.

### CODE REVIEW

File: docs/releases/1.10.1.txt [hunk 1]
```diff
@@ -14,3 +14,6 @@ Bugfixes
 
 * Allowed ``User.is_authenticated`` and ``User.is_anonymous`` properties to be
   compared using ``==`` and ``!=`` (:ticket:`26988`).
+
+* Fixed queryset filtering with ``isnull`` lookup on ``ForeignKey`` with its
```
Comment (timgraham) [inline docs/releases/1.10.1.txt]: Fixed the isnull lookup on a ForeignKey....
chop "argument"

**Comment by timgraham** [comment]
I think `queries` should be mostly complex queries that don't fit anywhere else. I don't know if it's worth it to try to find a better place for this new one (tests/many_to_one?). Up to you.

**Comment by claudep** [comment]
I thought that the ToFieldTests was a good home for that test. Maybe the whole ToFieldTests class could be moved to many_to_one at some point?

**Comment by timgraham** [comment]
Sounds like a good plan. I'm happy with anything to make "queries" less monolithic. :-)

**Pull request closed without merging**